definitions/contested/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -2512,6 +2526,20 @@
     "UserRole": "caseworker-divorce-financialremedy-courtadmin",
     "CRUD": "CRUD"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseFieldID": "allocatedToBeHeardAtHighCourtJudgeLevel",
+    "UserRole": "caseworker-divorce-financialremedy-courtadmin",
+    "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseFieldID": "allocatedToBeHeardAtHighCourtJudgeLevelText",
+    "UserRole": "caseworker-divorce-financialremedy-courtadmin",
+    "CRUD": "CRUD"
+  },
```

Comment:
I am not sure if the config for "caseworker-divorce-financialremedy-judiciary" is missing.

Reply:
the reason is that, "caseworker-divorce-financialremedy-judiciary" has the Authorisation setting in CaseEvent: "FR_newPaperCase", "FR_solicitorCreate" and "FR_amendApplication"

Reply:
After looking into this:
- A judge can't create a case so we don't expect to give the same rights as a caseworker/solicitor.
- It makes sense to give Read access for the judge so this will be added shortly.
- Further point, I've noticed that some fields don't have Read access for a judge which is confusing. Example is the reasonForLocalCourt field and the isApplicantsHomeCourt field. I wonder what the reason is for this.

Reply:
Read access for judge [41c1020](https://github.com/hmcts/finrem-ccd-definitions/pull/1915/commits/41c1020666f9a811cdba6b0134982b268bba78a6)